app/src/main/java/com/graymandev/sapp/model/Store.kt

```diff
@@ -0,0 +1,17 @@
+package com.graymandev.sapp.model
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class Store(
+    val id: Int,
+    val name: String,
+    val image: Int,
+    val score: Float,
```

Comment:
Don't use both Float and Double as they are not related to each other but do the same thing, I'd suggest making al non-integers Double